### ISSUE TITLE
Can you provide the resnet50+CBAM code with the pre-trained weight?

### ISSUE THREAD

**Issue by liudaizong** [Youngkl0726/Convolutional-Block-Attention-Module#3]
I want to compare the results with the SEblock, thx!

**Comment by Youngkl0726** [comment]
I can provide the pretrained weight, but the model I trained did not reach the accuracy mentioned in the paper, even worse than seresnet50.

**Comment by liudaizong** [comment]
Actually, i just want to use CBAM+resnet50 or SEblock+resnet50 for baseline. Do you have the seresnet50 code with pretrained weight on ImageNet in pytorch? The performance of seresnet50 in https://github.com/Cadene/pretrained-models.pytorch is not good enough.

**Comment by Rvv1296** [comment]
How can I add the CBAM into YOLOv4.
Can anyone suggest me the changes that I need to do in YOLOv4 code ?
Or anyone have pretrained weights for the YOLOv4 with CBAM ?
Thank you !!
